elkodon/src/lib.rs

```diff
@@ -278,9 +278,12 @@ pub mod service;
 
 /// Loads a meaninful subset to cover 90% of the elkodon communication use cases.
 pub mod prelude {
+    pub use crate::elk::Elk;
+    pub use crate::elk::ElkEvent;
     pub use crate::port::event_id::EventId;
     pub use crate::service::{
         process_local, service_name::ServiceName, zero_copy, Details, Service,
     };
+    pub use core::time::Duration;
```

Comment:
I'm don't know what is considered best practice for prelude but re-exporting std or core types might be something which the user would not expect. This also might easily break user code when `use elkodon::prelude::*` is used when the user already has a `use core::time::Duration` which is not that unlikely.

I understand the reason why you want to have it in the preloude, though.